test/run-tests

```diff
@@ -23,16 +23,64 @@ import urllib.parse
 import urllib.request
 
 import requests
+import cachecontrol
 import yaml
 
 HOSTNAME = socket.gethostname()
 
+transient_error_wait_time_default = 30
+transient_error_wait_time_max = 600
+transient_error_wait_time = transient_error_wait_time_default
+
+comment_cmd_test_all = "[citest]"
+comment_cmd_test_pending = "[citest pending]"
+comment_cmd_test_bad = "[citest bad]"
+
+
+def transient_error_wait_time_reset():
```

Comment:
IMHO it is better to have the verb at the beginning, i.e. `reset_transient_error_wait_time()`, it is more natural to read.